tests/qa_automation/qa_run.pm

```diff
@@ -69,12 +69,16 @@ sub qaset_config {
 # remove all existing zypper repos first
 sub prepare_repos {
     my $self           = shift;
-    my $qa_server_repo = get_var('QA_SERVER_REPO');
+    my $qa_server_repo = get_var('QA_SERVER_REPO', '');
+    my $qa_sdk_repo    = get_var('QA_SDK_REPO', '');
     pkcon_quit;
     if ($qa_server_repo) {
         # Remove all existing repos and add QA_SERVER_REPO
-        script_run('for ((i = 1; i <= $(zypper lr| tail -n+3 |wc -l); i++ )); do zypper -n rr $i; done; unset i', 300);
+        script_run('for ((i = $(zypper lr| tail -n+5 |wc -l); i >= 1; i-- )); do zypper -n rr $i; done; unset i', 300);
```

Comment:
I don't understand why you count repos starting with 5th. From the comment, do you wanna remove all repos here?
And please give verified run to prove.